app/src/main/java/np/com/naxa/factsnepal/splashscreen/SplashScreenActivity.java

```diff
@@ -23,11 +24,19 @@ protected void onCreate(Bundle savedInstanceState) {
             public void run() {
                 //Do something after 100ms
                 SharedPreferenceUtils sharedPreferenceUtils = new SharedPreferenceUtils(SplashScreenActivity.this);
-                if (sharedPreferenceUtils.getBoolanValue(LoginActivity.KEY_IS_USER_LOGGED_IN, false)) {
```

Comment:
You save data in shared pref in a line of code
 `SharedPreferenceUtils.getInstance(getContext()).getBoolanValue(LoginActivity.KEY_IS_USER_LOGGED_IN, false);`